### PR TITLE
289 failed call of ewiselambda

### LINKED ISSUE #289: Failed call of eWiseLambda
Added a test that fails on call of `eWiseLambda`:

https://github.com/Algebraic-Programming/ALP/blob/f2bd190839616067ce9be32ec1092a3fd5daf431/tests/unit/eWiseLambda.cpp#L53

I think that this line of code causes an issue:

https://github.com/Algebraic-Programming/ALP/blob/f2bd190839616067ce9be32ec1092a3fd5daf431/include/graphblas/reference/blas2.hpp#L2611

### PR DESCRIPTION
**Pull request by anyzelman**
Closes issue #289

### CODE REVIEW

File: tests/unit/eWiseLambda.cpp [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+static const size_t M_I[ 2 ] = { 8, 9 };
+static const size_t M_J[ 2 ] = { 9, 8 };
+
+// graphblas program
+void grbProgram( const void *, const size_t in_size, int &error ) {
+	error = 0;
+
+	if( in_size != 0 ) {
+		(void)fprintf( stderr, "Unit tests called with unexpected input\n" );
```
Comment (anyzelman) [inline tests/unit/eWiseLambda.cpp]: add space after `(void)`

File: tests/unit/eWiseLambda.cpp [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+    
+	grb::RC rc = buildMatrixUnique( M, &( M_I[ 0 ] ), &( M_J[ 0 ] ), M_val, 2, SEQUENTIAL );
+	if( rc != SUCCESS ) {
+		std::cerr << "\t initial buildMatrixUnique FAILED\n";
+		error = 5;
+	}
+
+	if( !error ) {
+        rc =  grb::eWiseLambda( [&M]( const size_t i, const size_t j, int& nz ) { nz = j; }, M );
```
Comment (anyzelman) [inline tests/unit/eWiseLambda.cpp]: missing tab (or spaces instead of tabs, not sure)

File: tests/unit/eWiseLambda.cpp [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+		error = 10;
+	}
+
+	if( !error ) {
+		for(auto it: M) {
+            if( it.second != it.first.second ) {
+                std::cerr << "\t eWiseLambda returned incorrect result\n";
+                error = 15;
+                break;
+            }
```
Comment (anyzelman) [inline tests/unit/eWiseLambda.cpp]: Spaces instead of tabs (should use tabs for indentation)

File: tests/unit/eWiseLambda.cpp [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+		}
+	}
+}
+
+int main( int argc, char ** argv ) {
+	(void)argc;
```
Comment (anyzelman) [inline tests/unit/eWiseLambda.cpp]: missing space before `argc`

**Review by anyzelman — COMMENTED** [review]
Only minor changes requested=)

**Comment by anyzelman** [comment]
Hi @aleksamilisavljevic -- I made an associated MR here, rebased on the latest `develop`, and did an initial review. Could you have a look and adapt the code accordingly?

**Review by aleksamilisavljevic — APPROVED** [review]

**Pull request merged**